CountriesV2/CountriesV2/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+using ConsoleTables;
+
+namespace CountriesV2
+{
+	class Program
+	{
+		static void Main(string[] args)
+		{
+			MyCsvReader reader = new MyCsvReader();
+			CountryDictionary countryDictionary = reader.ReadData("D:\\Countries.csv");
```

Comment:
change path to file to dynamic one. It will fail on the other system after cloning repository